src/sumlog.jl

```diff
@@ -0,0 +1,46 @@
+using IrrationalConstants: logtwo
+
```

Comment:
We already import the package, the convention here is to use explicit `IrrationalConstants.logtwo`.